superset-frontend/src/pages/CssTemplateList/index.tsx

```diff
@@ -196,7 +198,15 @@ function CssTemplatesList({
     subMenuButtons.push({
       name: (
         <>
-          <i className="fa fa-plus" /> {t('CSS template')}
+          <Icons.PlusOutlined
+            iconColor={theme.colors.primary.light5}
+            iconSize="s"
+            css={css`
+              margin: 'auto ${theme.gridUnit * 2}px auto 0';
+              vertical-align: 'baseline';
```

Comment:
I think you don't need to use quoting here. You might want to check others too

```suggestion
              margin: auto ${theme.gridUnit * 2}px auto 0;
              vertical-align: baseline;
```